src/indigo/less/sidebar.less

```diff
@@ -19,51 +19,48 @@
         cursor: pointer;
         font-size: 16px;
         font-weight: 400;
+        width: 100%;
 
-        > button {
+        button {
             padding: 3px 2px 0;
             border: none;
             background: #22262f;
-            color: @link-color;
+            color: @link-color !important;
             width: 99.5%;
             text-align: left;
+            font-size: 16px;
+
+
 
             &:focus {
                 outline: none;
+                 background: #22262F;
             }
 
             > span {
-                display: block;
+                    display: block;
```

Comment:
typo

Reply:
je to indentovany dost na hrubo. Nechtel jsem indentovat cely stylesheet abych se v tech dulezitych zmenach potom vyznal